content/100-getting-started/03-prisma-postgres/115-import-from-mysql-database.mdx

```diff
@@ -0,0 +1,346 @@
+  // delete-end
+  // add-start
+  provider = "postgres"
+  // add-end
+  url      = env("DATABASE_URL")
+}
+```
+
+In the generated `.env` file, update `DATABASE_URL` to match your new Prisma Postgres connection string that you received in [step 1](/getting-started/prisma-postgres/import-from-mysql-database#1-create-a-new-prisma-postgres-database):
+
+```terminal file=.env no-copy
+DATABASE_URL="prisma+postgres://accelerate.prisma-data.net/?api_key=__API_KEY__"
+```
+
+Introspect your newly migrated Prisma Postgres database and generate Prisma Client:
+
+```terminal
+npx prisma db pull
+```
+
+This command refreshes your Prisma models based on the new database schema.
+
+If you were using [Prisma Migrate](/orm/prisma-migrate/getting-started) before:
+
+- Delete your existing `migrations` folder in the `prisma` directory.
+- [Baseline your database](/orm/prisma-migrate/workflows/baselining#baselining-a-database) to begin creating new migrations.
+
+Congratulations! You've successfully migrated your MySQL database to Prisma Postgres and configured your Prisma project. Your migration tutorial is now complete.
+
+## Migrating data from PlanetScale to Prisma Postgres
+
+
+If you are migrating from a [PlanetScale](https://planetscale.com/) database, please ensure you have completed all the steps in this guide up to [step 1](/getting-started/prisma-postgres/import-from-mysql-database#1-create-a-new-prisma-postgres-database) before proceeding. This section will guide you through migrating your database schema and then migrating only the data.
+
+### 1. Migrate your database schema to Prisma Postgres
+
+Before moving your data, you need to transfer your existing PlanetScale schema to your new Prisma Postgres instance. If you have been using Prisma ORM with PlanetScale, you may not have a `migrations` folder (since you likely used `prisma db push`).
+
+If you aren’t using Prisma ORM yet, please follow the steps to set up Prisma with PlanetScale first:
+
+1. [Setup Prisma in your project](/getting-started/setup-prisma/add-to-existing-project/relational-databases-typescript-planetscale)
+2. [Introspect your PlanetScale database](/getting-started/setup-prisma/add-to-existing-project/relational-databases/connect-your-database-typescript-planetscale)
+
+Once your PlanetScale schema is available in your Prisma project, continue with the next steps.
+
+In your project `.env` file, update `DATABASE_URL` to match your new Prisma Postgres connection string that you received in the [step 1](/getting-started/prisma-postgres/import-from-mysql-database#1-create-a-new-prisma-postgres-database):
+
+```terminal file=.env no-copy
+DATABASE_URL="prisma+postgres://accelerate.prisma-data.net/?api_key=__API_KEY__"
+```
+
+In your `schema.prisma` file, modify the `provider` in the `datasource` block from `mysql` to `postgresql`:
+
+```prisma file=schema.prisma
+datasource db {
+  // delete-start
+  provider = "mysql"
+  // delete-end
+  // add-start
+  provider = "postgres"
```

Comment:
I know `postgres` also works but since we're using `postgresql` consistently in the docs I'd say we use it here to:

```suggestion
  provider = "postgresql"
```